scss/text.scss

```diff
@@ -17,6 +17,30 @@
   font-size: 12px;
 }
 
-.bold {
+.text-subtitle {
   font-weight: bold;
+  margin-bottom: 12px;
+}
+
+@media only screen and (max-width: 1024px) {
+  .text {
+    margin: 12px;
+    line-height: 1.3;
```

Comment:
Set the line height in each `text-` class, cause this prop depends on the font-size